tests/robustness/coverage/README.md

```diff
@@ -0,0 +1,123 @@
+## Overview
+
+This is still Work In Progress.
+
+Go tests in this directory analyze the usage of Etcd API based on collected
+traces from a running cluster. They will output information on:
+
+1. Traces that **did go** through the contract interface:
```

Comment:
This list seems like it will get outdated quickly. I also think the first value should be classification of requests done by apiserver. Information whether requests goes through interface should be just part of it.